src/components/ListingDetail/ListingDetail.js

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Card from '../UI/Card/Card';
+import CardTitle from '../UI/CardTitle/CardTitle';
+
+const ListingDetail = (props) => {
+  const [title] = useState('Listing Description');
+  const [description] = useState(
+    "Elevator possibilities. Additional space can be added with elevator. Once inside this compound you don't know you're in the city. Multiple gardens encasing the property are so dense you can't see beyond the trees. New Construction 2020. Perfection is the only word to describe this stunning property. Ideal for the urban family who wants a city lifestyle but demands an outdoor experience that is breathtaking. Pool capable. The rear terrace has a Wolf Grill,Sub-Zero and an ashlar stonewall that exudes quality. Heated garage has a 100 amp Tesla charging station,wifi,cameras & alarms. Second floor has a 600 sq foot heated Ipe private deck with 25 trees in custom cedar planters,cameras and wifi. Unlimited hyper luxury interior customizations with high-end features. Both Radiant & Hydro heat in 20 rooms,5 wolf ovens,3 Sub-Zeros,3 washers/dryers,2 dishwashers,all 4 utilities added,new sewer,2 inch water line,4 inch gas line,400 amps electric buried. Nothing compares in MA!!"
+  );
+
+  return (
+    <Card className={`listing-detail ${props.className ? props.className : ''}`}>
+      <CardTitle className="listing-detail__title" title={title} />
```

Comment:
I feel like CardTitle is too granular of a component. I would just pass the title in to the Card component.

As for the className, how we handle this in map-view is that we have a prop called `passedClass` (Don't like the name personally. `classBlock` would probably be better) that we use to pass down the block part of the class. Then the component uses that block part and adds the element parts to come up with all the class names.

Then you could then just add your component like this:
```
<Card classBlock="listing-detail" title={title}>
  {description}
</Card>
```

In this example I also took out the className that was passed in to this component because it doesn't really seem necessary for this component to have two class names, one called `aside__listing-detail` and one called `listing-detail`. Typically when a new block starts, it just gets its own class and drops the parent block name.